test/common.js

```diff
@@ -51,19 +51,17 @@ function rmdirSync(p, originalEr) {
   }
 }
 
-function refreshTmpDir() {
-  if (!process.send) { // Not a child process
-    try {
-      rimrafSync(exports.tmpDir);
-    } catch (e) {
-    }
+exports.refreshTmpDir = function() {
+  try {
+    rimrafSync(exports.tmpDir);
+  } catch (e) {
+  }
```

Comment:
I think this usually goes on the same line (`} catch (e) { }`, is the linter OK with that?